javascript/javascript3/week1/Giphy.js

```diff
@@ -0,0 +1,29 @@
+const apiKey = "MNafT5n97vrsAEtVj4iE7QSBGpb1F08e";
```

Comment:
api keys are secret keys that shouldn't pushed to the github repository. Your repository is public and anyone can use them. Normally there is an .env and .gitignore files where you keep your hidden data. Don't worry, you will learn more in Node class. But just keep in mind never push your api keys value or passwords or sensitive data to your repository

Reply:
Ok Diny. Thank you so much for your feedback :)